### DISCUSSION TITLE
Why doesn't the `Task.Delay` method have `Async` suffix?

### DISCUSSION THREAD

**Discussion by kiminuo** [dotnet/runtime#59761] [Q&A] [answered]
I wonder why `Task.Delay` is not called `Task.DelayAsync`. Was this inconsistency introduced intentionally or not?

Thank you

**Comment by stephentoub** [comment, marked as answer]
Because Task itself is about asynchrony and so it's deemed duplicative... effectively the "async" is built into the type, e.g. Task.Run isn't Task.RunAsync.  Only when there's ambiguity, e.g. we added a synchronous Wait() method, have we used an Async suffix, e.g. WaitAsync.
